### ISSUE TITLE
all what we have to do is the following instead of using a pod

### ISSUE THREAD

**Issue by AymanOmara** [tristanhimmelman/HidingNavigationBar#83]
func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
        let translation = scrollView.panGestureRecognizer.translation(in: scrollView.superview)
        if(translation.y > 0){
            self.navigationController?.setNavigationBarHidden(false, animated: true)
        }else{
            self.navigationController?.setNavigationBarHidden(true, animated: true)
        }
        
    }

**Comment by mrmatil** [comment]
Yeah, especially when we for ex. have to implement tabbar hiding and complex animation for it.
Useless issue
